src/pretalx/schedule/forms.py

```diff
@@ -248,6 +248,7 @@ def __init__(self, *args, **kwargs):
             "Information for speakers, e.g.: Projector has only HDMI input."
         )
         self.fields["capacity"].widget.attrs["placeholder"] = "300"
+        self.fields["guid"].widget.attrs["placeholder"] = self.instance.uuid or ""
```

Comment:
The idea was to be able to differentiate if the guid was filled in externally or not, thereby I currently only fill the guid field in the database if the user explicitly entered it.

Reply:
I can see the intention, but a) I don't think it really makes a difference, and b) setting the placeholder might be the most frustrating solution in that case, because it will *show* the generated UUID to the user, but it won't allow the user to copy it, thereby forcing them to either copy it manually or head into the page source code in order to retrieve it.

Reply:
Endusers never should need to copy the guid from the form field: 

* You either import rooms with their guid via schedule.json/xml – and if someone change their name the downstream system can still identify the smae room by its guid. 
* The other case is when room was already assigned an ID in another system, then you can set the guid field to that existing value.

Reply:
"Should" never need to copy it, yes. But if the data is there, the user "should" be able to retrieve it by a different way than downloading a weird XML export, or copying by sight, or going to browser tools.

I still feel like this would not be a problem if we just always generated the `guid` (but allowed the user to change it). Am I overlooking a critically useful aspect of always computing it when it's not explicitly chosen by the user?